src/components/multi-select/multi-select.css

```diff
@@ -0,0 +1,86 @@
+@import '../../styles/colors-v2.css';
+@import '../../styles/spacing.css';
+@import "../../styles/typography-v2.css";
+
+.container {
+  composes: tag-list from '../tag-list/tag-list.css';
+  height: auto;
+  margin: var(--spacing-small) 0;
```

Comment:
AFAIK, this shouldn't have any margin. This is a screenshot of our MDS Input:

![image](https://user-images.githubusercontent.com/841084/119703794-c55c2480-be0b-11eb-9194-b5b48201cc7b.png)


This is the screenshot of the MDS MultiSelect:

![image](https://user-images.githubusercontent.com/841084/119703836-d016b980-be0b-11eb-8ce4-bd62985ba29f.png)

Reply:
Actually, this raises a concern:

- Do we want to make the whole component's container the "tag list"? I would figure the "tag list" is actually the "form control children container" layer.

Reply:
I've removed the extra margin here.

Re your concern: I think it makes sense from a compositional standpoint (and also from a layout/styling standpoint) to keep the `FormControlIcons` and children out of the `TagList`. They're not tags, they don't represent state that is directly derived from specific options, and they need to stay in the top-right-hand corner of the "input", in a different flex container.